.github/workflows/build-tag-publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Checkout project sources
         uses: actions/checkout@v4
         with:
-          ref: ${{ github.event.pull_request.merge_commit_sha }}
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
can this change be pursued independently?

Reply:
Yes.